web_tool_script_3.R

```diff
@@ -218,7 +223,8 @@ create_interactive_report(
   ipr_results_stress_test = ipr_results_stress_test,
   display_currency = display_currency,
   currency_exchange_value = currency_exchange_value,
-  header_dictionary = header_dictionary
+  header_dictionary = header_dictionary,
+ sector_order = sector_order
```

Comment:
My suggestion above adds logic that you can't inject in a variable like `sector_order()`, but you can inject it in a function `sector_order()`.

Reply:
Let me try something out. An example may be clearer than my words above.